sentry-android-okhttp/src/main/java/io/sentry/android/okhttp/SentryOkHttpInterceptor.kt

```diff
@@ -39,6 +41,7 @@ class SentryOkHttpInterceptor(
             }
             throw e
         } finally {
+            span?.apply(beforeSpan)
```

Comment:
Is there a way we can drop the span? Similar to how we do `beforeSend` and `beforeBreadcrumb`?

We would also need to pass some context to the callback, like the request (and resposne?) objects.

Some context: We're trying to add a way for a user to do:

```
if (request.url == "/health" || response.status == 200) {
return null;
```

Reply:
Good point about dropping the span. url and status can be obtained from span but in less convenient way than with plain request and response. I'll change the implementation.